src/sentry/static/sentry/app/views/releasesV2/utils/index.tsx

```diff
@@ -26,3 +31,16 @@ export const convertAdoptionToProgress = (
   percent: number,
   numberOfProgressUnits = 5
 ): number => Math.ceil((percent * numberOfProgressUnits) / 100);
+
+type ProgressRingColorFn = React.ComponentProps<typeof ProgressRing>['progressColor'];
+export const getCrashFreePercentColor: ProgressRingColorFn = ({percent, theme}) => {
+  if (percent < CRASH_FREE_DANGER_THRESHOLD) {
+    return theme.red;
+  }
+
+  if (percent < CRASH_FREE_WARNING_THRESHOLD) {
+    return theme.yellowOrange;
+  }
+
+  return theme.green;
```

Comment:
Are we going to run into colour blindness issues using a red, orange, green colour scheme?

Reply:
@markstory we are probably going to replace this component with one that shows crash free rate in a less linear way - because this number should most of the time be in 90s (hopefully even 99.xxx) and it's not ideal to visualize those small differences this way... don't know about the color blindness issues though, maybe @doralchan?